### DISCUSSION TITLE
Sharing a SharedWorker between domains

### DISCUSSION THREAD

**Discussion by fl0rek** [eigerco/lumina#311] [Ideas]
Lumina instance running in a SharedWorker can be accessed only by pages coming from the same domain (browser policy, due to security reasons). 

Could use an iframe coming from our domain loaded by pagees on other domains to act as a proxy and run single global Lumina instance? Does it have security implications we need to keep in mind (now or in the future)?
Maybe browser extension instead?

**Comment by zvolin** [comment]
will they be able to access the same idb storage using iframe?
